### ISSUE TITLE
Move `mdformat` back into @hukkin's account

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/meta#1238]
@hukkin requested that we move the `mdformat` package back into his GitHub account, here's the request:

> I'd really like to see this happen (given I keep dictatorship over mdformat (it can be less draining than shared governance)), or alternatively transfer mdformat (and any plugins you wish to give up) back to my account. It worries me a bit that there is no response here in almost 2 years.
>
> Mdformat was transferred here mostly because years ago Chris S. happened to ask and I couldn't see much of a downside. Having it under the same organization as the parser it uses seemed reasonable. I was never well aware (still today am not 😄 ) of this organization's structure, funding, governance, product, values, or anything really, except for the fact that Chris S. had ported markdown-it under this org which was useful to me.
>
> I feel today, for my maintenance, this organization is mostly a burden, setting false expectations of governance and processes. A few times I've noticed I don't have the permissions to do what I wanted. I often feel unsure about my rights to manage other repositories besides mdformat, even if I have the commit bit. Often having a clear single owner would've made things simpler.

### Chris' take

I think it makes sense to move the mdformat project back out into @hukkin's account. It wasn't created as part of the grant, and he's the original and current maintainer.

I suggest that @rowanc1 @jstac or @gregcaporaso chime in if they have any thoughts or objections. If not, then I'll plan to move the repository in a few days.

### Will anything break?

Are there any considerations we need to take in doing the actual moving? E.g., things that might break?

**Comment by hukkin** [comment]
Thanks @choldgraf

I'll also gladly take any direct or indirect dependencies of `mdformat-gfm`, which out of EBP owned mdformat plugins seems to be only `mdformat-tables`. But @chrisjsewell is the author so I think that's for him to decide.

Regarding other EBP owned mdformat plugins, if there's someone to maintain them here (or elsewhere), that'd be great. If not (or I'm the expected maintainer), I'd rather have them transferred too.

**Comment by choldgraf** [comment]
yeah for all the mdformat stuff I'm +1 on moving that to your (or some other) org

**Comment by hukkin** [comment]
Friendly ping @choldgraf 

In case you transfer `mdformat-tables` to me too, please also do so on PyPI (my username is the same as here).

> I'd ask that you respect Chris' wishes if he wants them moved under his account and communicates that to you later on.

Of course!

**Comment by choldgraf** [comment]
I've moved `mdformat` to my account and initiated a transfer to @hukkin - can you accept? (apparently this is the only way to move an org repo like this).

I've done the same for `mdformat-tables` but I need @hukkin to delete your fork first so that I can transfer.

For PyPI it'll be a bit more time because I am temporarily locked out of my account. Perhaps @rowanc1 can do this part of the transfer if he has any cycles.

**Comment by hukkin** [comment]
Thanks @choldgraf, I've accepted `mdformat` and removed `mdformat-tables` fork.

**Comment by chrisjsewell** [comment]
> @chrisjsewell if you prefer mdformat-tables to go under your account

yeh sure it's fine to go under @hukkin 's

It would be nice if one day #841 could be addressed, as @hukkin notes, this organisation now a burden for any development

**Comment by KyleKing** [comment]
Hi, I'm adding a +1 to this issue. There is some work on `mdformat-tables` to support TOML configuration, which would be nice to release, but currently blocked by the transfer of the PyPi package (https://github.com/hukkin/mdformat-tables/issues/31)

**Comment by choldgraf** [comment]
I tried to do this a few weeks ago, and was blocked on these three issues that are linked together:

- https://github.com/executablebooks/meta/issues/1252
- https://github.com/pypi/support/issues/5347
- https://github.com/executablebooks/meta/issues/1251
